SampleApp/DemoMapViewController.swift

```diff
@@ -97,9 +98,18 @@ class DemoMapViewController:  SampleMapViewController, MapMarkerSelectDelegate {
       "Spanish": "es_ES",
       "Korean": "ko_KR",
       "Italian": "it_IT",
+      "OSM Default": "none",
       ]
     let actionSheet = UIAlertController.init(title: "Map Language", message: "Choose a language", preferredStyle: .actionSheet)
     for (actionTitle, languageIdentifier) in languageIdByActionSheetTitle {
+      if (languageIdentifier == "none") {
+        actionSheet.addAction(UIAlertAction.init(title: actionTitle, style: .default, handler: { [unowned self] (action) in
+          let update = TGSceneUpdate(path: "global.ux_language", value: "")
```

Comment:
Isn't that the job of the SDK method `updateLocale`? We could rather make it accept an empty locale or a `nil` locale and fill up the `return` statement [here](https://github.com/mapzen/ios/blob/d0e60fc0207ba25853297770b3809c4412cdda84/MapzenSDK/MZMapViewController.swift#L622) to update the global ux language to en empty string, that would give a good default behavior.